framework/infrastructure/S3/__test__/s3ObjectStore.test.ts

```
@@ -0,0 +1,217 @@
+                .mockResolvedValueOnce({ Body: makeBody('data-2') })
+                .mockResolvedValueOnce({ Body: makeBody('data-3') });
+
+            const store = makeStore();
+            const streams = await store.streamDownloadBatchObjects([
+                'key-1',
+                'key-2',
+                'key-3'
+            ]);
+
+            expect(streams).toHaveLength(3);
+            streams.forEach((s) => expect(s).toBeInstanceOf(Readable));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)